### PR TITLE
GSYE-295: Add settings for day-ahead market

### PR DESCRIPTION
**Pull request by hannesdiedrich**

### CODE REVIEW

File: gsy_framework/constants_limits.py [hunk 1]
```diff
@@ -247,6 +248,7 @@ class GlobalConfig:
     DURATION_D = 1
     MARKET_COUNT = 1
     FUTURE_MARKET_DURATION_HOURS = 0
+    DAY_AHEAD_DURATION_DAYS = 1
```
Comment (fievelk) [inline gsy_framework/constants_limits.py:251]: Here I'm always not sure of the difference between `GlobalSettings` and `ConstSettings.FutureMarketSettings`. Could you remind me why we don't put both in the same class?

**Review by fievelk — APPROVED** [review]
LGTM with one question

File: gsy_framework/constants_limits.py (hunk 1 above)
Comment (hannesdiedrich) [reply gsy_framework/constants_limits.py:251]: Good question! I always forget. Having a look at the docstring does not really help either. 
I think I just copied the behaviour from the futures markets' FUTURE_MARKET_DURATION_HOURS. Which does not conform to the above comment `Default simulation settings d3a-web side` ( I will exchange d3a with gsy, too) either.
 
I propose to move both FUTURE_MARKET_DURATION_HOURS and FUTURE_MARKET_DURATION_HOURS into ConstSettings. FUTURE_MARKET_DURATION_HOURS. What do you think?

Comment (fievelk) [reply gsy_framework/constants_limits.py:251]: That works for me 👍 Thanks!

**Comment by hannesdiedrich** [comment]
I am closing this PR because the design of the day-ahead market was changed. As soon as the design is finialised, we will update the ticket and I will create a new PR.

**Pull request closed without merging**